tests/selection.spec.ts

```diff
@@ -522,13 +522,20 @@ test('click the list icon can select and copy', async ({ page }) => {
   await initThreeLists(page);
   await assertRichTexts(page, ['123', '456', '789']);
   await clickListIcon(page, 0);
+  // copy 123
```

Comment:
I'm not sure about the expected behavior here, since when pasting in the previous test case, the editor doesn't have native focus. @QiShaoXuan hope for further refinement, would be great if you can track this comment in some clipboard-related issue.

/cc @JimmFly @lawvs